extensions/3DTILES_metadata/README.md

```diff
@@ -0,0 +1,115 @@
+
+![3DTILES_metadata Spec Map](figures/spec_map.jpg)
+
+## Concepts
+
+### Metadata
+
+Metadata refers to application specific information. It can be associated with different components of 3D Tiles: tileset, layers and tiles.
+
+#### Classes
+
+Classes serve as the templates for the metadata objects - they provide a list of properties and the type information for those properties. For example, a tileset containing different layers of 3D data might create classes for each type:
+
+```javascript
+"classes": {
+  "DatasetClass": {
+    "properties": {
+      //...
+    }
+  },
+  "PhotogrammetryClass": {
+    "properties": {
+      //...
+    }
+  },
+  "BIMClass": {
+    "properties": {
+      //...
+    }
+  },
+  "PointCloudClass": {
+    "properties": {
+      //...
+    }
+  }
+}
+```
+
+This extension uses the classes in compliance with the [Cesium 3D Metadata Specification](https://github.com/CesiumGS/3d-tiles/tree/3d-metadata-spec/specification/Metadata#classes).
+
+#### Properties
+
+Each class provides a list of properties. A property has a `type` and, for the array types, it may include a `componentType` and a `componentCount`. Additionally, a property may be designated as `optional`, and if so, a `default` value for the property may be provided to apply to all instances of the class that do not set a value for the property. For example, a property of `ARRAY` type may provide an empty array as the `default`. To learn more about properties, refer to the [Cesium 3D Metadata Specification](https://github.com/CesiumGS/3d-tiles/tree/3d-metadata-spec/specification/Metadata#classes).
+
+```javascript
+"classes": {
+  "PhotogrammetryClass": {
+    "properties": {
+      "sensorVersion": {
+        "type": "STRING"
+      }
+    }
+  },
+  "BIMClass": {
+    "properties": {
+      "modelAuthor": {
+        "type": "STRING"
+      }
+    }
+  },
+  "PointCloudClass": {
+    "properties": {
+      "scanner": {
+        "type": "STRING"
+      }
+    }
+  }
+}
+```
+
+
+#### Tileset Metadata
+
+The tileset metadata object may specify a `name` and `description` to display the information in a user interface. The tileset metadata object may also conform to a `class` and assign values to the `properties` defined by the selected class. The tileset object must use the [single instance shorthand syntax](https://github.com/CesiumGS/3d-tiles/tree/3d-metadata-spec/specification/Metadata#single-intance-shorthand) to assign values to its properties.
+
+```javascript
+"extensions": {
+  "3DTILES_metadata": {
+    "classes": {
+      "dataset": {
+        "properties": {
+          "author": {
+            "type": "STRING"
+          }
+        }
+      }
+    },
+    "tileset": {
+      "class": "dataset",
+      "properties": {
+        "author": "Cesium"
+      }
+    }
+  }
+}
```

Comment:
Maybe add a bit more variation to the example, `dataset` is a bit generic and it would be nice to see multiple different types of properties.